### PR TITLE
[fixbuild] Update to Node 16

### PR DESCRIPTION
**Pull request by somewhatabstract**
## Summary:
Did this while investigating an issue and thought we should keep it since we are targetting Node 16 now.

Issue: XXX-XXXX

## Test plan:
`yarn test`
`yarn build`
Check the actions all work as expected.

### CODE REVIEW

File: packages/wonder-stuff-i18n/package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
         "access": "public"
     },
     "engines": {
-        "node": ">=12"
+        "node": ">=16"
```
Comment (jeresig) [inline packages/wonder-stuff-i18n/package.json:6]: Why do only some packages have a node 16 restriction?

**Review by jeresig — APPROVED** [review]

File: packages/wonder-stuff-i18n/package.json (hunk 1 above)
Comment (somewhatabstract) [reply packages/wonder-stuff-i18n/package.json:6]: I don't know the answer to that.

Comment (somewhatabstract) [reply packages/wonder-stuff-i18n/package.json:6]: I can modify it so they all do

Comment (jeresig) [reply packages/wonder-stuff-i18n/package.json:6]: Well, either all do or none do - whichever!

**Pull request merged**